### PR TITLE
Fix takedowns

### PR DESCRIPTION
**Pull request by firyx**
# PULL REQUEST

## Overview

This PR fixes 2 takedown issues:
- `portal-setup-following` with enabled deploy dodn't honor portal takedown
- portal takedown run only `sia` docker service, not `mongo`

Moreover this PR enables portal deploys and restarts to takedown portals.

## Checklist
Review and complete the checklist to ensure that the PR is complete before assigned to an approver.
 - [x] All new methods or updated methods have clear docstrings
 - [x] Testing added or updated for new methods
 - [x] Verify if any changes impact the WebPortal Health Checks
 - [x] Approriate documentation updated
 - [x] Changelog file created

### CODE REVIEW

File: playbooks/portals-takedown.yml [hunk 1]
```diff
@@ -42,15 +42,17 @@
       include_tasks: tasks/portal-docker-compose-files-get-wanted.yml
 
     # Start just sia docker service
-    - name: Start just sia docker service
+    - name: Start just mongo and sia docker services
       community.docker.docker_compose:
         project_src: "{{ webportal_dir }}"
         files: "{{ webportal_docker_compose_files_wanted }}"
         build: False
```
Comment (kwypchlo) [inline playbooks/portals-takedown.yml:49]: does it mean we're not rebuilding ? I think this should be set to true ?

**Review by MSevey — DISMISSED** [review]

File: playbooks/portals-takedown.yml (hunk 1 above)
Comment (firyx) [reply playbooks/portals-takedown.yml:49]: On takedown playbook we are not rebuilding, we just start existing mongo and sia services. Similar on restart playbooks, we are not rebuilding on them. We are rebuilding on deploys.

Comment (kwypchlo) [reply playbooks/portals-takedown.yml:49]: ok but we still can run setup-portal following and it will update the stack but start only sia and mongo right?

Comment (firyx) [reply playbooks/portals-takedown.yml:49]: Yes, if the portal is in takedown group.

**Review by peterjan — APPROVED** [review]
LGTM 👍

**Pull request merged**